### PR TITLE
Ingnore vendor/ directory

### PR DESCRIPTION
**Pull request by m1guelpf**
Fixes https://github.com/librariesio/bibliothecary/issues/371

- [x] Have you followed the guidelines for [contributors](http://docs.libraries.io/)?
- [x] Have you checked to ensure there aren't other open pull requests on the repository for a similar change?
- [x] Is there a corresponding ticket for your pull request?
- [x] Have you written new tests for your changes?
- [x] Have you successfully run the project with your changes locally?

If so then please replace this section with a link to the ticket(s) it addressed, an explanation of your change and why you think we should include it. Thanks again!

### CODE REVIEW

**Comment by andrew** [comment]
@m1guelpf you'll also need to update the test here: https://github.com/librariesio/bibliothecary/blob/master/spec/bibliothecary_spec.rb#L60

**Comment by m1guelpf** [comment]
@andrew I'm on it. On mobile you can only edit one file at a time :wink:

**Comment by andrew** [comment]
Cool, heading out, will merge and release a new version later today 👍

File: spec/bibliothecary_spec.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@
   end
 
   it 'ignores certain files' do
-    expect(described_class.ignored_files_regex).to eq(".git|node_modules|bower_components|spec/fixtures|vendor/bundle|dist")
+    expect(described_class.ignored_files_regex).to eq(".git|node_modules|bower_components|spec/fixtures|dist")
```
Comment (andrew) [inline spec/bibliothecary_spec.rb]: @m1guelpf can you add back in `vendor|` here to fix the tests

**Comment by m1guelpf** [comment]
@andrew Done!

**Pull request merged**